packages/web-service/src/pages/declaration/declaration.njk

```diff
@@ -4,13 +4,12 @@
 {% set title = "Declaration" %}
 
 {% block pageContent %}
-<p class="govuk-body">By submitting this application you are hereby declaring this information is upto date and correct to the best of your knowledge. </p>
-{{ govukButton({
-    attributes: { id: "continue" },
-    preventDoubleClick: true,
-    name: 'continue',
-    text: buttonText if buttonText else 'Confirm',
-    classes: "govuk-!-margin-top-5",
-    href: "/submission"
```

Comment:
Moved this functionality to the setData handler in the POST, as that it is pattern